src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -491,15 +498,23 @@ public void testAddFieldsWithReservedNames() throws Exception
         DomainFieldRow clientFieldWarning = domainFormPanel.addField("select * from table");
```

Comment:
This test name is inaccurate. We're testing more than just reserved field names.
Something like `testFieldNameErrors` would be more accurate.

Reply:
I like it.